blockchains/eth/lib/withdrawals_decoder.js

```diff
@@ -0,0 +1,36 @@
+const { computeGasExpense, computeGasExpenseBase36 } = require('./util');
+const constants = require('./constants');
+
+class WithdrawalsDecoder {
+  constructor(web3, web3Wrapper) {
+    this.web3Wrapper = web3Wrapper;
+  }
+
+  getWithdrawal(withdrawal, block, blockNumber) {
+    return [{
+      from: constants.ETH_WITHDRAWAL,
+      to: withdrawal.address,
+      value: this.web3Wrapper.parseHexToNumber(withdrawal.amount),
+      valueExactBase36: this.web3Wrapper.parseValueToBN(withdrawal.amount).toString(36),
+      blockNumber: blockNumber,
+      timestamp: this.web3Wrapper.parseHexToNumber(block.timestamp),
+      transactionHash: 'WITHDRAWAL_${blockNumber}',
+      type: 'beacon_withdrawal'
+    }];
+  }
+
+
+  getBeaconChainWithdrawals(block, blockNumber) {
+    const result = [];
+    block.withdrawals.forEach((withdrawal) => {
+      //const blockNumber = this.web3Wrapper.parseHexToNumber(block.number);
```

Comment:
I'm guessing this is being parsed in ETH_WORKER already, so that's why you've commented it out?